Automattic/wp-calypso
# BBE In Progress Page: switch to dedicated endpoint

Fixes https://github.com/Automattic/martech/issues/2016

## Proposed Changes

This PR switches to the dedicated BBE endpoint for showing content submission status on Calypso. This addresses the issue where the status was not shown correctly on atomic sites due to sporadic delays in options syncing.

## Testing Instructions



* Go to `/start/do-it-for-me`.
* Choose "Existing Site" and then select an existing atomic site which has a `.wpcomstaging.com` address.
* Go through the steps and complete the purchase.
* After purchase, go to `/home/<site slug>`.
* Confirm that you see a CTA to submit content.
* Click on the CTA and fill out the content form.
* After submission, confirm that you are redirected to `/home/<site slug>` and the page says that the content has been submitted.

## Pre-merge Checklist
